scalameta/semanticdb-scalac-core/src/main/scala/scala/meta/internal/semanticdb/InputOps.scala

```diff
@@ -41,6 +42,10 @@ trait InputOps { self: DatabaseOps =>
   }
 
   implicit class XtensionGPositionMPosition(pos: GPosition) {
+    def toSchema: s.Position = {
+      val mpos = toMeta
+      s.Position(mpos.start, mpos.end)
+    }
     def toMeta: m.Position = {
```

Comment:
Do we still need `toMeta` here and in other places?

Reply:
we shouldn't, these methods should be removed.